WhaleSpotting/Models/DbModels/SightingDbModel.cs

```diff
@@ -20,5 +20,7 @@ public class SightingDbModel
         public OrcaType? OrcaType { get; set; }
         public string OrcaPod { get; set; }
         public bool Confirmed { get; set; }
+
```

Comment:
Could you add a comment with TODO here pls :)

Reply:
done